airflow/providers/apache/hdfs/provider.yaml

```diff
@@ -48,3 +48,6 @@ hooks:
   - integration-name: WebHDFS
     python-modules:
       - airflow.providers.apache.hdfs.hooks.webhdfs
+
+hook-class-names:
+  - airflow.providers.apache.hdfs.hooks.hdfs.HDFSHook
```

Comment:
What about webhdfs?

Reply:
> I'd like us to delay importing hooks if we can -- I think it will slow things down? WDYT @mik-laj

Yeah. I think this makes sense. That would help with the snowflake SSL problem as well. Will look into it

Reply:
> What about webhdfs?

I only added those hooks/connections that either were there already, or had custom fields. We can have a follow up change and see which of the remaining hooks we want to add ? WDYT @ashb ?

Reply:
Oh I thought this was _was_ the follow up :) Sorry. ETOOMANYPRs